mspy_vendi/api/v1/geography.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, Depends
+from fastapi.responses import ORJSONResponse
+
+from mspy_vendi.core.api import CRUDApi, basic_endpoints, basic_permissions
+from mspy_vendi.core.enums import ApiTagEnum
+from mspy_vendi.core.pagination import Page
+from mspy_vendi.deps import get_db_session
+from mspy_vendi.domain.geographies.schemas import GeographyDetailSchema
+from mspy_vendi.domain.geographies.service import GeographyService
+
+router = APIRouter(prefix="/geography", default_response_class=ORJSONResponse, tags=[ApiTagEnum.GEOGRAPHIES])
+
+
+class GeographyAPI(CRUDApi):
+    service = GeographyService
+    schema = GeographyDetailSchema
+    current_user_mapping = {**basic_permissions}
```

Comment:
```suggestion
    current_user_mapping = basic_permissions
```

Reminer: basic_permissions is mutable, that's why maybe I need to use .copy() for it. Will analyze